bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/ESH-INF/thing/airthings.xml

```diff
@@ -14,8 +15,6 @@
 		<description>Indoor air quality monitor with radon detection</description>
 
 		<channels>
-			<channel id="rssi" typeId="rssi"/>
```

Comment:
Does it harm to have it listed here? If Things are created manually, this has the advantage that this channel is available without any handler required to be present.

Reply:
I didn't realize they had that use case. I'll add them back in then.